hack/Buchsbaum_frequency.py

```diff
@@ -0,0 +1,107 @@
+import numpy as np
+import astropy.units as u
+from plasmapy.formulary.frequencies import gyrofrequency, plasma_frequency
+from plasmapy.particles import Particle
+from plasmapy.utils.decorators import (
+    angular_freq_to_hz,
+    validate_quantities,
+)
+__all__ = ["buchsbaum_frequency"]
+__aliases__ = ["w_bb_", "w_ii_", "w_bi_"]
+__all__ += __aliases__
+
+@validate_quantities(
```

Comment:
```suggestion
__all__ += __aliases__


@validate_quantities(
```

2 blank lines before a function or class definition